articles/custom-domains/index.md

```diff
@@ -164,6 +165,20 @@ If you want to use social identity providers with your custom domain, you must u
 You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
 :::
 
+#### SAML identity provider configuration
+
+If you want to use SAML identity providers with your custom domain, you must obtain the service provider metadata from Auth0 (such as `https://login.northwind.com/samlp/metadata?connection=CONNECTION_NAME`). This includes updated Assertion Consumer Service (ACS) URLs. You will then need to manually update this value in your IdP(s). This change to your IdP(s) will need to happen at the same time as you enable your custom domains. This can pose a problem if there are multiple IdPs to configure.
+
+Alternatively, you can use signed requests to fulfill this requirement. Once your custom domain is set up, in the [Dashboard](${manage_url}/#/tenant/custom_domains), you will need to download the certificate from the link under the “Sign Request” toggle. You can then give the certificate to the IdP(s) to upload. This enables the IdP to validate the signature on the AuthnRequest message that Auth0 sends to the IdP. The certificate needs to be imported into the IdP and, if necessary, signature verification should be enabled (exact steps vary by IdP).
+
+You should then turn on the “Sign Request” toggle in the Dashboard under **Connections > Enterprise > SAMLP > CONNECTION**. This will trigger Auth0 to sign the SAML AuthnRequest messages it sends to the IdP.
```

Comment:
All of this detail about signing requests seems like too much detail for the custom domains doc. It seems like we should include this in the `saml-sp-generic` doc (about setting up SAML connections), and link to it from here. "If you're using signed requests (link), the ACS  url will be sent in the signed request"... wdyt?

Reply:
I feel like this section is pretty specifically talking about using custom domains + signed connections. I suppose we could generalize it, but with this being in a separate feature-specific doc now, maybe that's enough to get it "out of the way" for other custom domain use cases?